### PR TITLE
Expose Animation::value_track_interpolate to GDscript

### PR DESCRIPTION
**Pull request by Novakasa**
`bezier_track_interpolate` and `transform_track_interpolate` are already exposed, so not sure why this was missing.
I added the entry to the class reference.

This is my first PR, so I hope I did everything right.

### CODE REVIEW

**Comment by Arrow-x** [comment]
Congrats on your first commit, this is very useful thank you

File: doc/classes/Animation.xml [hunk 1]
```diff
@@ -681,6 +681,17 @@
...
 		</method>
+		<method name="value_track_interpolate" qualifiers="const">
+			<return type="float">
+			</return>
+			<argument index="0" name="track_idx" type="int">
+			</argument>
+			<argument index="1" name="time" type="float">
+			</argument>
+			<description>
+				Returns the interpolated value at the given [code]time[/code] (in seconds). The [code]track_idx[/code] must be the index of a value Track.
```
Comment (akien-mga) [inline doc/classes/Animation.xml:692]:
```suggestion
				Returns the interpolated value at the given [code]time[/code] (in seconds). The [code]track_idx[/code] must be the index of a value track.
```

File: doc/classes/Animation.xml [hunk 2]
```diff
@@ -681,6 +681,17 @@
 				Sets the update mode (see [enum UpdateMode]) of a value track.
 			</description>
 		</method>
+		<method name="value_track_interpolate" qualifiers="const">
+			<return type="float">
+			</return>
+			<argument index="0" name="track_idx" type="int">
+			</argument>
+			<argument index="1" name="time" type="float">
```
Comment (akien-mga) [inline doc/classes/Animation.xml:689]: This should be `time_sec` according to the method binding.

**Comment by akien-mga** [comment]
Note that the email you used to sign the commit seems to be attributed to @novakasa and not @lolligerjoj. It's not a problem per se, but mentioning it in case it's not your own account and you'd want to change it.

**Comment by lolligerjoj** [comment]
Yeah, I think I will fix this, have two different systems with different git settings there

**Review by akien-mga — APPROVED** [review]

**Comment by akien-mga** [comment]
Thanks!

**Comment by akien-mga** [comment]
Cherry-picked for 3.2.4.

**Pull request merged**